### PR TITLE
fix: web feed link 404 when set custom base

### PR DESCRIPTION
**Pull request by uWayLu**
**Summary**

**What kind of change does this PR introduce?** (check at least one)

- [x] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Docs
- [ ] Build-related changes
- [ ] Other, please describe:

If changing the UI of default theme, please provide the **before/after** screenshot:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)

You have tested in the following browsers: (Providing a detailed version will be better.)

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [ ] IE

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature
- [ ] Related documents have been updated
- [ ] Related tests have been updated

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

File: components/Feed.vue [hunk 1]
```diff
@@ -29,6 +29,11 @@ export default {
       }
       return false
     },
+    getFeedFilePathWithBase()
```
Comment (billyyyyy3320) [inline components/Feed.vue:32]: Thanks for your contribution!

There's already been a computed function getting the feed path. Please whether remove it or modify it.
Besides, you can wrap the path with `$withBase`. Ref: https://vuepress.vuejs.org/guide/assets.html#base-url

**Review by billyyyyy3320 — CHANGES_REQUESTED** [review]

File: components/Feed.vue (hunk 1 above)
Comment (uWayLu) [reply components/Feed.vue]: Thanks for the information! I modified it.

**Review by billyyyyy3320 — APPROVED** [review]

**Comment by mariomka** [comment]
When is a new release that includes it planned?

Thank you

**Comment by billyyyyy3320** [comment]
@mariomka Check out [v2.3.3](https://github.com/vuepress/vuepress-theme-blog/releases/tag/v2.3.3)

**Pull request merged**